packages/block-editor/src/hooks/color.scss

```diff
@@ -4,10 +4,6 @@
 	.block-editor-contrast-checker {
 		grid-column: span 2;
 		margin-top: $grid-unit-20;
-
-		.components-notice__content {
-			margin-right: 0;
-		}
```

Comment:
This is the ContrastChecker component, and this style is no longer needed.

<img width="512" height="334" alt="Image" src="https://github.com/user-attachments/assets/4982de23-d102-4ecf-b3e9-9522c13ad66e" />